pynocular/database_model.py

```diff
@@ -186,6 +231,21 @@ def initialize_table(cls, table_name: str, database_info: DBInfo) -> None:
                 type = sqlalchemy_uuid()
             elif field.type_ is datetime:
                 type = TIMESTAMP(timezone=True)
+            elif field.type_.__name__ == "NestedModel":
```

Comment:
?
```suggestion
            elif isinstance(field.type_, NestedModel):
```

Reply:
Ill try that but `NestedModel` is a dynamically generated one so idk if the linter will get the reference

Reply:
No worries if it doesnt work

Reply:
just confirmed it cant find `NestedModel`